src/runtime/operatormethod.cs

```diff
@@ -86,7 +104,9 @@ public static void FixupSlots(IntPtr pyType, Type clrType)
             Debug.Assert(_opType != null);
             foreach (var method in clrType.GetMethods(flags))
             {
-                if (!IsOperatorMethod(method))
+                // We don't want to override slots for either non-operators or
+                // comparison operators, which are handled by ClassBase.tp_richcompare.
+                if (!IsOperatorMethod(method) || IsComparisonOp(method))
```

Comment:
NIT:  `!OpMethodMap .ContainsKey(method)`

Reply:
Of course, thanks!